backend/pipelined/filter.go

```diff
@@ -5,35 +5,27 @@ import (
 	"context"
 	"time"
 
+	"github.com/sensu/sensu-go/asset"
+	"github.com/sensu/sensu-go/js"
 	"github.com/sensu/sensu-go/types"
-	"github.com/sensu/sensu-go/util/eval"
+	"github.com/sensu/sensu-go/types/dynamic"
 	utillogging "github.com/sensu/sensu-go/util/logging"
 )
 
-func evaluateEventFilterStatement(event *types.Event, statement string) bool {
+func evaluateJSFilter(event interface{}, expr string, assets asset.RuntimeAssetSet) bool {
```

Comment:
Is it strictly necessary to change `event` from being a `*types.Event` to being a `interface{}`?